content/en/workflows/build.md

```diff
@@ -3,7 +3,126 @@ title: Build Workflows
+            "parameters": [
+                {
+                    "name": "teamId",
+                    "value": "ABC1234"
+                },
+                {
+                    "name": "channel",
+                    "value": "#workflows-test"
+                },
+                {
+                    "name": "mentionTargets",
+                    "value": [
+                        "@Bits"
+                    ]
+                },
+                {
+                    "name": "text",
+                    "value": "Hello world!"
+                }
+            ]
+        }
+    ],
+    "startStepName": "Send_message",
+    "connectionEnvs": [
+        {
+            "env": "default",
+            "connections": []
+        }
+    ]
+}
+{{< /code-block >}}
+
+## Context variables
+
+Creating useful workflows sometimes necessitates passing data from one step to another, or configuring steps that act on data that comes from the workflow's trigger source. You can perform this kind of data interpolation with context variables.
+
+Context variables come in three varieties:
+- A small collection of standard **workflow variables** are present in all workflows.
+- Some steps come with built-in **step output variables** that allow you to pass data from that step to a subsequent step in your workflow.
+- **Trigger variables** are passed into the workflow by the triggering event.
+
+The **Context Variables** tab for each step provides a map of all context variables available to that step.
+
+{{< img src="workflows/context-variables.png" alt="The Context Variables tab" >}}
+
+Access a context variable in a step by enclosing it in double braces (`{{`). Context variables are available in fields marked with the `{{` notation.
+{{< img src="workflows/use-context-variable.mp4" alt="Use double fences in a supported text field to insert a context variable" video="true" >}}
+
+### Workflow variables
+
+All workflows have three standard variables:
+- `WorkflowName`: The name of the workflow. Accessed with `{{ WorkflowName }}`.
+- `WorkflowId`: The workflow ID. Accessed with `{{ WorkflowId }}`.
+- `InstanceId`: Each workflow run receives a unique instance ID. Access the instance ID with `{{ InstanceId }}`.
+
+### Step output variables
+
+Some steps create outputs that are available to subsequent steps in a workflow. Access a step variable with the syntax: `Steps.<step_name>.<variable>`. For example, to retrieve the pull request status variable (`state`) from the GitHub pull request status step (`Get_pull_request_status`), you'd use the following context variable:
+
+```
+{{ Steps.Get_pull_request_status.state }}
+```
+
+### Trigger variables
+
+You can pass trigger variables into a workflow as inputs. Workflows accept a JSON object of comma-separated key-value pairs. Access a trigger variable in workflow steps using the syntax `{{ Trigger.key }}`. For example, to access the trigger variable `{ "user": "Bits" }`, use ` {{ Trigger.user }}` in the step.
```

Comment:
Same thing as above - is there a way to get a list of the available trigger variables for a step?